www/caddy/src/opnsense/service/conf/actions.d/actions_caddy.conf

```
@@ -0,0 +1,29 @@
+[start]
+command:/usr/local/opnsense/scripts/Pischem/Caddy/caddy_control.py start
+parameters:
+type:script_output
```

```suggestion
type:script
```

this shouldn't return anything because it's never echoed